### ISSUE TITLE
Possible to refine training in production?

### ISSUE THREAD

**Issue by theophile** [dedupeio/dedupe#891]
Apologies in advance if this isn't an appropriate place for a question like this. I'm using the dedupe library to maintain an ever-growing database of names and addresses from numerous possible sources with numerous possible variations. I periodically run a method to create match clusters, which I then go through manually to decide whether the DB entries are the same and should be merged, or whether they're distinct and should remain separate.

As part of this, if the cluster is just a pair of two entries, then I have the function add to the training.json file depending on whether the pair is a match or if it's distinct.

However, if I'm understanding the code correctly, the deduper only reads from the settings file, not the training.json file, meaning that this data being added to the json file isn't actually being used the next time the deduper runs. I think I can run the trainer again and then immediately "finish," and it will write a new settings file that includes the additional data from the updated json file. But is there a more automatic way to do this? 

Basically, I want to continue to "teach" the deduper in this manner, without having to run new training sessions. Is there a good way to do this?
